dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -51,6 +51,14 @@ cookie.secure.desc=Specifies that the session cookies include the secure field.
 cookie.http.only=Include HttpOnly field in session cookies
 cookie.http.only.desc=Specifies that session cookies include the HttpOnly field. Browsers that support the HttpOnly field do not enable cookies to be accessed by client-side scripts. Using the HttpOnly field will help prevent cross-site scripting attacks.
 
+# The word SameSite should not be translated
+cookie.same.site=Include SameSite attribute in session cookies
```

Comment:
**change**
Include SameSite attribute in session cookies
**to**
Include a SameSite attribute in session cookies

unless "the" is more appropriate than "a"